Modules/Core/Common/include/itkImageRegion.h

```diff
@@ -275,8 +275,8 @@ class ITK_TEMPLATE_EXPORT ImageRegion final : public Region
   bool
   IsInside(const Self & otherRegion) const
   {
-    const auto otherIndex = otherRegion.m_Index;
-    const auto otherSize = otherRegion.m_Size;
+    const auto & otherIndex = otherRegion.m_Index;
+    const auto & otherSize = otherRegion.m_Size;
```

Comment:
Both `Index` and `Size` are trivially copyable (having a super-fast compiler-generated copy-constructor and an empty  compiler-generated destructor). When declaring otherIndex and otherSize as reference, each access to the data referred to by those variables (as in the for-loop below) will have an extra indirection. Both otherIndex[i] and otherSize[i] are accessed multiple times. That's why I wonder: is this really a performance improvement?

Reply:
I understood this as a change to silence a warning.

Reply:
IMHO, _otherRegion_ object is already passed by constant reference to the function. Not sure that there are some "extra indirection" costs for _otherIndex_ and _otherSize_, IMHO they are just aliases, but maybe you are right. Just close the PR if you think there are some costs for "extra indirection" of constant reference.